### PR TITLE
docs: Correct xform descriptions in class_basis

### PR DESCRIPTION
**Pull request by mdraw**

### CODE REVIEW

**Comment by tagcup** [comment]
There are two "return" verbs in the new description

File: doc/base/classes.xml [hunk 1]
```diff
@@ -7211,7 +7211,7 @@
 			<argument index="0" name="v" type="Vector3">
 			</argument>
 			<description>
-				Return a vector transformed (multiplied) by the transposed matrix and return it. Note that this is a multiplication by inverse only when the matrix represents a rotation-reflection.
+				Return a vector transformed (multiplied) by the transposed matrix. Note that this is only a multiplication by the inverse of the matrix if it represents a rotation-reflection.
```
Comment (bojidar-bg) [inline doc/base/classes.xml]: I think it should be `only when` rather than `only .... if`, as it used to be, as it is clearer (IMHO).

**Review by bojidar-bg — APPROVED** [review]
Looks pretty good.

**Comment by mdraw** [comment]
I am not a native speaker, but I thought "if" has to be used here because "when" can only be used in a temporal sense, see http://dictionary.cambridge.org/grammar/british-grammar/if-or-when

**Comment by bojidar-bg** [comment]
@mdraw I'm not a native speaker either, so I can't say for sure. Anyway, you might make it `only if` as well (since we want to say that "this will happen **only in this case**", not that "**only this** will happen")

**Comment by akien-mga** [comment]
Merged manually with a slight edit based on above discussion as 070384c9c.

**Pull request closed without merging**